File: problemas/matematica/1585/1585.cs
using System;

class URI {
    static void Main(string[] args) {
        int N = int.Parse(Console.ReadLine());
        for(int i = 0; i < N; ++i){
            string[] entrada = Console.ReadLine().Trim().Split(' ');

            int x = int.Parse(entrada[0]);
            int y = int.Parse(entrada[1]);

            Console.WriteLine($"{(x * y) / 2} cm2");
        }
    }
}

File: problemas/matematica/1585/1585.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let n: i32 = input.trim().parse().unwrap();

    for _ in 0..n {
        input = String::new();
        io::stdin().read_line(&mut input);
        let parts: Vec<&str> = input.split(" ").collect();
        let x: i32 = parts[0].trim().parse().unwrap();
        let y: i32 = parts[1].trim().parse().unwrap();

        println!("{} cm2", (x * y) / 2)
    }
}
